app/component/ItineraryList.js

```diff
@@ -98,7 +98,7 @@ function ItineraryList(
         ) || 'rail';
       const legs = itineraries.flatMap(itinerary => itinerary.legs);
       const showBikeBoardingInfo = config.showBikeBoardingInfoHeader?.(
-        config.bikeBoardingInfoModes,
+        config.bikeBoardingModes,
```

Comment:
This is somewhat strange logic. Notifier is decided by looking a single mode of first transit leg.